seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/exception/JdbcConnectorErrorCode.java

```diff
@@ -27,8 +27,9 @@ public enum JdbcConnectorErrorCode implements SeaTunnelErrorCode {
     CONNECT_DATABASE_FAILED("JDBC-04", "Connector database failed"),
     TRANSACTION_OPERATION_FAILED(
             "JDBC-05", "transaction operation failed, such as (commit, rollback) etc.."),
-    NO_SUITABLE_DIALECT_FACTORY("JDBC-06", "No suitable dialect factory found");
-
+    NO_SUITABLE_DIALECT_FACTORY("JDBC-06", "No suitable dialect factory found"),
+    DONT_SUPPORT_SINK("JDBC-07", "The jdbc type don't support sink"),
+    KERBEROS_AUTHENTICATION_FAILED("JDBC-08", "Kerberos authentication failed");
```

Comment:
Can you help me update this exception file: https://github.com/apache/seatunnel/blob/dev/docs/en/connector-v2/Error-Quick-Reference-Manual.md

Reply:
yes